delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/pages/UserEditPage.html

```diff
@@ -28,6 +28,7 @@
         div.wicket-tree-table{
             margin: 10px 0;
             width: 100%;
+            color: black;
```

Comment:
```suggestion
            color: var(--sakai-text-color-1);
```
would be my preference.
edit: but I see how that doesn't work since the component background remains white. I am ok with hard-coding this black until this component is refactored.

Reply:
Yeah, this component needs additional work I created a new issue for that. It just seemed like the quick fix of making the text black at least would make it *usable* in dark mode, even though it's not very bright.